src/main/java/jpa/domain/linestation/LineStation.java

```diff
@@ -0,0 +1,37 @@
+package jpa.domain.linestation;
+
+import jpa.domain.base.BaseEntity;
+import jpa.domain.line.Line;
+import jpa.domain.station.Station;
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AttributeOverride(name = "id", column = @Column(name = "lineStationId"))
```

Comment:
BaseEntity의 id라는 컬럼 이름이 override 되어서 lineStationId로 지정될 것 같아요.
데이터베이스 컬럼 이름 규칙과 조금 맞지 않는 것 같네요

```suggestion
@AttributeOverride(name = "id", column = @Column(name = "line_station_id"))
```

Reply:
말씀하신대로 BaseEntity의 id를 EmbbedId형태로 override해서 사용하는게 가능할 지 모르겠군요 ㅠㅠ
아래와 같이는 되지 않을까 싶은데, 지금 구조에서는 조금 힘들지 않을까 싶네요

```java
@Getter
@MappedSuperclass
@EntityListeners(AuditingEntityListener.class)
public class BaseGenericEntity<T> {
    private T id;
    // ...
}
```

이 부분은 저도 한번 찾아보도록 하겠습니다.
혹시 좋은 방법을 알게되면 공유해주셔도 좋을 것 같네요! 👍

Reply:
앗😱 적용해보다가 삭제한다고 했는데 남아있었네요. 제거하겠습니다! 
 
`제네릭`은 생각못했는데 알려주신 방법으로도 고민해보겠습니다. 꼼꼼한 리뷰 감사합니다 🙇